todocore/src/main/java/com/knighttodo/todocore/rest/DayResource.java

```diff
@@ -10,12 +10,10 @@
 import com.knighttodo.todocore.rest.request.DayRequestDto;
 import com.knighttodo.todocore.rest.response.DayResponseDto;
 import com.knighttodo.todocore.service.DayService;
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.ApiResponse;
-import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.*;
```

Comment:
remove wildcard please, you can set it up in Intellij Idea to not use * until there are more than 10 imports from the same packages

Reply:
Thank you. I found this fileld in settings idea.